db/main.go

```diff
@@ -104,11 +104,26 @@ func createGoogleDriveClient(googleDriveCredentialsFile, goodleDriveOAuthTokenFi
 
 func summarize(source, state string, datastoreClient *datastore.Client, googleDriveService *drive.Service) error {
 	query := fmt.Sprintf("name contains '%s' and '%s' in parents", state, source) // pegando os arquivos com prefixo 'estado' da pasta de id 'source'
-	fileList, err := googleDriveService.Files.List().Q(query).Do()
-	if err != nil {
-		return fmt.Errorf("falha ao buscar arquivos do estado [%s] no diretório [%s], erro %q", state, source, err)
+	var result *drive.FileList
+	var setToResolve []*drive.File
+	var err error
+	for {
+		if result != nil && result.NextPageToken == "" {
+			break
+		}
```

Comment:
Completamente opcional

```suggestion
	for ; result == nil ||  result.NextPageToken != " ; {
```